BUTTON_A : 0
BUTTON_B : 1
BUTTON_X : 2
BUTTON_Y : 3
BUTTON_LB : 4
BUTTON_RB : 5
BUTTON_BACK : 6
BUTTON_START : 7

L_JOY_HORIZ : 0
L_JOY_VERT : 1
L_TRIGGER : 2
R_JOY_HORIZ : 3
R_JOY_VERT : 4
R_TRIGGER : 5
DPAD_HORIZ : 6
DPAD_VERT : 7